app/notifications/utils.py

```diff
@@ -0,0 +1,8 @@
+import requests
+
+
+def confirm_subscription(confirmation_request):
+    url = confirmation_request['SubscribeURL']
+    response = requests.get(url)
+    response.raise_for_status()
+    return confirmation_request['TopicArn']
```

Comment:
We log it on the [caller](https://github.com/alphagov/notifications-api/pull/920/files#diff-ff083df1227c89e7812cac1d7c48bb4aR38)